client-service/src/main/java/com/telecominfraproject/wlan/client/validator/ClientSessionValidatorException.java

```diff
@@ -0,0 +1,11 @@
+package com.telecominfraproject.wlan.client.validator;
+
+public class ClientSessionValidatorException extends Exception
```

Comment:
Explicit checked exceptions (as opposed to the descendants of runtime ones) are polluting the code with "throws" declarations and/or mandatory try-catch blocks, which usually just re-throw those exceptions. I think it is cleaner to make this a descendant of runtime exception or DsDataValidationException, and let it break the whole API request and let the caller deal with it.

Reply:
I see your point and I will change this class to extend DsDataValidationException, since it will be nice and clean